CONTRIBUTING.md

```diff
@@ -36,6 +36,18 @@ Contributor Agreement (ECA) on file.
 For more information, please see the Eclipse Committer Handbook:
 https://www.eclipse.org/projects/handbook/#resources-commit
 
+## Project Contribution Guidelines
+
+ * Code should adhere to the style defined at https://github.com/eclipse-ee4j/ee4j.
+ * Contributions should be made via github pull requests at https://github.com/eclipse-ee4j/servlet-api/pulls 
+ * Pull Requests should have reviews requested from several project committers from different organisations than the author(s). 
+ * Pull Request by committers that receive no review responses after a reasonable time, considering time zones and working days, can be interpreted as approved.
+ * Pull Requests should pass continuous integration tests prior to merging.
+ * Pull Requests for significant contributions, new features or changed behaviour should be open for comment and review in a non draft state for at least 1 week prior to merging.
+ * For disputed Pull Requests, any committer may request a formal consensus vote (ie. +1, 0, -1). Any such vote will be scheduled, conducted and adjudicated by the Project Leader(s).
```

Comment:
May add something like "... attempts should first be made to resolve issues through discussion. If consensus can't be reached after a reasonable period of time then...". Of course, that raises the question "How long is reasonable?". I'd probably leave that vague as I think it will vary from case to case.

Reply:
I put in the Scheduled by project leader bit so they could allow a reasonable time for discussion in case somebody was too quick to call a vote.   But I agree the guidelines should indicate that formal votes are exceptional. How about:
+ In exceptional cases when a consensus on a Pull Request cannot be established through normal review and discussion, then any committer may request a formal consensus vote (ie. +1, 0, -1). Any such vote will be scheduled, conducted and adjudicated by the Project Leader(s).